phoenix-core/src/it/java/org/apache/phoenix/end2end/SystemTablesCreationOnConnectionIT.java

```diff
@@ -81,27 +88,27 @@
     private static final String PHOENIX_NAMESPACE_MAPPED_SYSTEM_CATALOG = "SYSTEM:CATALOG";
     private static final String PHOENIX_SYSTEM_CATALOG = "SYSTEM.CATALOG";
     private static final String EXECUTE_UPGRADE_COMMAND = "EXECUTE UPGRADE";
-    private static final String MODIFIED_MAX_VERSIONS ="5";
+    private static final String MODIFIED_MAX_VERSIONS = "5";
     private static final String CREATE_TABLE_STMT = "CREATE TABLE %s"
-            + " (k1 VARCHAR NOT NULL, k2 VARCHAR, CONSTRAINT PK PRIMARY KEY(K1,K2))";
+        + " (k1 VARCHAR NOT NULL, k2 VARCHAR, CONSTRAINT PK PRIMARY KEY(K1,K2))";
```

Comment:
These line continuations do match the conventions in https://phoenix.apache.org/contributing.html .
I know that we have code in much worse shape than this, but since you are fixing formatting, we might as well do it by the book.

Reply:
True, i just realized that i had continuation tab as `4` as opposed to `8`.

Reply:
you meant `do not` match the conventions above right?

Reply:
https://phoenix.apache.org/develop.html details the steps for Eclipse and intelliJ